compute/admin_guide/configure/proxy.adoc

```diff
@@ -42,9 +42,8 @@ A number of settings let you specify how Prisma Cloud interfaces with your proxy
 [.section]
 ==== Proxy bypass
 
-You can provide a list of addresses that Prisma Cloud can contact directly without connecting through the proxy.
-Specify DNS names, IP addresses, or a combination of both. 
-Specifying IP addresses in CIDR notation is supported. Specifying DNS names using wildcards is supported. 
+You can provide a list of addresses—DNS names, IP addresses, or a combination of both—that Prisma Cloud can contact directly without connecting through the proxy.
+Specifying IP addresses in CIDR notation is supported. Specifying DNS names using wildcards is supported.
```

Comment:
@pfox1969 , can we rephrase this 
 CIDR notation or <what is the other one subnet mask>format  are supported for IP addresses, and DNS names support the use of wildcards.